test/NameRegistry.t.sol

```diff
@@ -2621,131 +2625,345 @@ contract NameRegistryTest is Test {
+            assertEq(nameRegistry.recoveryClockOf(tokenIds[i]), 0);
+        }
     }
 
-    function testReclaimBiddableNames(address alice, address mod, address recovery) public {
-        _assumeClean(alice);
+    function testReclaimBiddableNames(
+        address[4] calldata users,
+        address mod,
+        address recovery,
+        address[4] calldata destinations
+    ) public {
         _assumeClean(mod);
         _assumeClean(recovery);
+        address[] memory addresses = new address[](10);
+        for (uint256 i = 0; i < users.length; i++) {
+            _assumeClean(users[i]);
+            _assumeClean(destinations[i]);
+            addresses[i] = users[i];
+            addresses[i + 4] = destinations[i];
+        }
+        addresses[8] = mod;
+        addresses[9] = recovery;
+        _assumeUnique(addresses);
+
+        bytes16[] memory fnames = new bytes16[](4);
+        fnames[0] = "alice";
+        fnames[1] = "bob";
+        fnames[2] = "carol";
+        fnames[3] = "dan";
+
+        uint256[] memory tokenIds = new uint256[](4);
+        tokenIds[0] = ALICE_TOKEN_ID;
+        tokenIds[1] = BOB_TOKEN_ID;
+        tokenIds[2] = CAROL_TOKEN_ID;
+        tokenIds[3] = DAN_TOKEN_ID;
+
+        for (uint256 i = 0; i < fnames.length; i++) {
+            _register(users[i], fnames[i]);
+        }
 
-        _register(alice);
         uint256 biddableTs = block.timestamp + REGISTRATION_PERIOD + RENEWAL_PERIOD;
         _grant(MODERATOR_ROLE, ADMIN);
-        _requestRecovery(alice, recovery);
+
+        for (uint256 i = 0; i < fnames.length; i++) {
+            _requestRecovery(users[i], tokenIds[i], recovery);
+        }
 
         vm.warp(biddableTs);
-        vm.expectEmit(true, true, true, true);
-        emit Transfer(alice, POOL, ALICE_TOKEN_ID);
+        NameRegistry.ReclaimAction[] memory reclaimActions = new NameRegistry.ReclaimAction[](4);
+
+        for (uint256 i = 0; i < users.length; i++) {
+            reclaimActions[i] = NameRegistry.ReclaimAction(tokenIds[i], destinations[i]);
+            vm.expectEmit(true, true, true, true);
+            emit Transfer(users[i], destinations[i], tokenIds[i]);
+        }
         vm.prank(ADMIN);
-        nameRegistry.reclaim(ALICE_TOKEN_ID);
+        nameRegistry.reclaim(reclaimActions);
 
         // reclaim should extend the expiry ahead of the current timestamp
         uint256 expectedExpiryTs = block.timestamp + RENEWAL_PERIOD;
 
-        if (alice != POOL) assertEq(nameRegistry.balanceOf(alice), 0);
-        assertEq(nameRegistry.balanceOf(POOL), 1);
-        assertEq(nameRegistry.ownerOf(ALICE_TOKEN_ID), POOL);
-        assertEq(nameRegistry.expiryOf(ALICE_TOKEN_ID), expectedExpiryTs);
-        assertEq(nameRegistry.recoveryOf(ALICE_TOKEN_ID), address(0));
-        assertEq(nameRegistry.recoveryClockOf(ALICE_TOKEN_ID), 0);
+        for (uint256 i = 0; i < users.length; i++) {
+            assertEq(nameRegistry.balanceOf(users[i]), 0);
```

Comment:
nit: can we maintain the same ordering of these checks across all test methods? it makes it a lot easier to review when scanning many lines.  here we check expiryOf before ownerOf, but in other places we do the opposite